react-frontend/src/components/StyleComponents/card.js

```diff
@@ -154,14 +154,18 @@ export const CommunityCardStyled = styled(Link).attrs({
   className: 'cardRound',
 })`
   background: #003366;
-  border-radius: ${cardBorderRadius};
+  border-radius: ${cardBorderRadius} ${cardBorderRadius} 30px 30px;
   height: 95%;
   margin-bottom: 20px;
   width: 100%;
   display: flex;
   flex-direction: column;
   :hover {
-    box-shadow: 5px 7px grey;
+    border: 5px solid #fdb917;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    position: relative;
```

Comment:
Sorry for suggesting these styles in the first place without fully testing....I see that they make the UI jumpy...

Could you please remove `position`, `right`, and `left` properties, but add these properties:

```
transition: 0.25s;
transition-timing-function: ease-in-out;
```
Additionally, for the `:focus` state, could you add:
```
outline: -webkit-focus-ring-color auto 5px !important;
```
👆Reason for this: for Accessibility, we need to get the outline back to displaying when tabbing through the links on the page. 

👇These styles are in the code somewhere and are causing the outline not to display when the card is in `:focus` when tabbing through links on the page:
```
a:active, a:focus, a:hover {
 text-decoration: none;
 outline: 0;
}
```